sphinx/builders/xml.py

```diff
@@ -71,6 +71,7 @@ def write_doc(self, docname: str, doctree: Node) -> None:
         # work around multiple string % tuple issues in docutils;
         # replace tuples in attribute values with lists
         doctree = doctree.deepcopy()
+        doctree.document.attributes["xmlns:py"] = "https://github.com/sphinx-doc/sphinx/issues/9413"
```

Comment:
I think it's not good to set the URL of #9413. I'd like to use a better one for this. What kind of URL should we use instead? document? project page (GitHub)?

Reply:
Maybe we can point to the documentation of the domains?

Reply:
We can point only the URL of the "bundled" domains: https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html
But this document does not contains the 3rd party domains. For example, sphinx-http-domain: https://pypi.org/project/sphinx-http-domain/.

Reply:
Maybe we can keep it as an attribute in the domain class with default pointing to the "bundled" domains doc?